modules/currency.js

```diff
@@ -81,12 +78,9 @@ function initCurrency(url) {
   conversionCache = {};
   currencySupportEnabled = true;
 
-  if (!originalBidResponse) {
-    utils.logInfo('Installing addBidResponse decorator for currency module', arguments);
+  utils.logInfo('Installing addBidResponse decorator for currency module', arguments);
 
-    originalBidResponse = bidmanager.addBidResponse;
-    bidmanager.addBidResponse = addBidResponseDecorator(bidmanager.addBidResponse);
-  }
+  bidmanager.addBidResponse.addHook(addBidResponseHook, 100);
```

Comment:
is there any value in setting a priority when no hooks compete yet?

Reply:
Yes.  If others add hooks later this one will still run first.  I specifically wanted currency running first as it buffers responses (if the currency file isn't back yet).